### PR TITLE
Add changelog endpoint

### PR DESCRIPTION
**Pull request by andrcuns**
Adds changelog entry generation endpoint introduced in gitlab 13.9: https://docs.gitlab.com/ee/api/repositories.html#generate-changelog-data

### CODE REVIEW

File: spec/gitlab/objectified_hash_spec.rb [hunk 1]
```diff
@@ -13,8 +13,10 @@
     let(:oh) { described_class.new(hash) }
 
     it 'allows to call Hash methods' do
-      expect(oh['foo']).to eq('bar')
-      expect(oh.merge(key: :value)).to eq('foo' => 'bar', key: :value)
+      capture_output do
+        expect(oh['foo']).to eq('bar')
+        expect(oh.merge(key: :value)).to eq('foo' => 'bar', key: :value)
+      end
```
Comment (NARKOZ) [inline spec/gitlab/objectified_hash_spec.rb:19]: why is this change necessary?

Comment (andrcuns) [reply spec/gitlab/objectified_hash_spec.rb:19]: I can remove this. Those expect statements are printing some stuff to stdout which is inserted in the middle of rspec test output slightly messing up the visuals, but definitely not strictly necessary if you feel this is unnecessary.

At least I think I added it because of that, it's been a while.

Comment (andrcuns) [reply spec/gitlab/objectified_hash_spec.rb:19]: Yes, as the commit message says `Silence warning in test`

Comment (NARKOZ) [reply spec/gitlab/objectified_hash_spec.rb:19]: Let's revert it

**Comment by andrcuns** [comment]
Should be good now, removed all unrelated changes.

**Review by NARKOZ — APPROVED** [review]

**Comment by NARKOZ** [comment]
Thanks!
